apps/web/public/static/locales/zh-CN/common.json

```diff
@@ -2169,5 +2169,208 @@
   "email_team_invite|content|invited_to_regular_team": "{{invitedBy}} 已邀请您加入他们在 {{appName}} 上的团队“{{teamName}}”。{{appName}} 是一个活动安排调度程序，让您和您的团队无需通过电子邮件沟通即可安排会议。",
   "privacy": "隐私",
   "cookie_consent_checkbox": "我同意 Cal.com 的隐私政策以及 cookie 使用政策",
-  "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ 在此上方添加您的新字符串 ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑"
```

Comment:
The script isn't respecting this separator. This should be always at last.

Reply:
Oops, didn't notice that. Solved now!

Reply:
Didn't notice that, solved now.